module/motion/KinematicsConfiguration/data/config/KinematicsConfiguration.yaml

```diff
@@ -26,8 +26,8 @@ head:
     base_position_from_origin: [-0.007, 0, 0.21] # [x, y, z]
   # head movement limits
   limits: # pitch limit is a function of yaw ... need to calculate this properly
-    yaw: [-pi/4, pi/4] # [min_yaw, max_yaw]
-    pitch: [pi/6, pi/6] # [min_pitch, max_pitch]
+    yaw: [-pi/3, pi/3] # [min_yaw, max_yaw]
+    pitch: [-pi/5, pi/5] # [min_pitch, max_pitch]
```

Comment:
Could you please explain this change?

Reply:
This is needed to allow the head to move